decidim-core/app/forms/decidim/base_form.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+module Decidim
+  # A base form object to hold common logic, like automatically adding as
+  # attributes the params sent as context by the `FormFactory` concern.
+  class BaseForm < Rectify::Form
```

Comment:
Sorry to be so picky, but what about just `Form`? xD

Also, why do you need to include `current_organization` and `current_user` as attributes? Sounds like me like this could be a potential security vulnerability, in case you send the attributes from the view (they would be overriden!).